Documentation/tutorial/intro_toc.md

```diff
@@ -47,49 +58,64 @@ Three kinds of links are supported:
-*items*       | *TOC Item Object* | Specifies the children *TOC Items* of current *TOC Item*.
+Property Name | Required or Optional | Type              | Description
+------------- | -------------------- | ----------------- | ---------------------------
+*name*        | required             | string            | Specifies the title of the *TOC Item*.
+*href*        | optional             | string            | Specifies the navigation path of the *TOC Item*. If *href* is not specified, the *TOC Item* serves as a container to parent its children *TOC Items*. If *href* is an absolute path, the *TOC Item* navigates to that specified path. If *href* is a relative path, see [below](#href-in-detail) for details.
+*items*       | optional             | *TOC Item Object* | Specifies the children *TOC Items* of current *TOC Item*.
 
-**Advanced**: These properties are useful when a TOC links another TOC, or links to a uid.
+**Advanced usages**: Following properties are useful when a TOC links to another TOC, or links to a uid. They are all optional.
 
 Property Name                     | Type              | Description
 --------------------------------- | ----------------- | ---------------------------
-*topicHref*                       | string            | Specifies the topic href of the *TOC Item*. It is useful when *href* is linking to a folder or *tocHref* is used.
-*topicUid*                        | string            | Specifies the `uid` of the *topicHref* file. If the value is set, it overwrites the value of *topicHref*.
+*topicHref*                       | string            | Specifies the topic href of the *TOC Item*. It is useful when *href* is linking to a folder or TOC file or *tocHref* is used.
+*topicUid*                        | string            | Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *topicHref*.
 ~~*homepage*~~ **Deprecated**     | string            | ~~Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.~~ Use *topicHref* instead.
 ~~*uid*~~ **Deprecated**          | string            | ~~Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*.~~ Use *topicUid* instead.
 ~~*homepageUid*~~ **Deprecated**  | string            | ~~Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.~~ Use *topicUid* instead.
 
-Href in detail
----------------
-If a *TOC Item* is linking to some relative path, there are three cases:
+## *href* in detail
+
+If a *TOC Item* is linking to a relative path, there are three cases:
 
 1. Linking to another *TOC File*, for example, `href: examples/toc.md`.
 2. Linking to a folder, which means, the value of the link ends with `/` explicitly, for example, `href: examples/`
 3. Linking to some local file.
 
-Each case is described in detail below.
+Each case is described in details below.
```

Comment:
>in details [](start = 23, length = 10)

in detail?
[https://english.stackexchange.com/questions/23986/in-detail-vs-in-details](https://english.stackexchange.com/questions/23986/in-detail-vs-in-details)

Reply:
nice

---
In reply to: [208491924](https://github.com/dotnet/docfx/pull/3165#discussion_r208491924) [](ancestors = 208491924)